app/models/load_profile.rb

```diff
@@ -9,6 +9,11 @@ class LoadProfile < ActiveRecord::Base
 
   validates :key, presence: true, uniqueness: true
 
+  has_attached_file :curve, styles: {
```

Comment:
Did you mean to add this back in? I understood this had moved into LoadProfileComponent?